rhalphalib/util.py

```diff
@@ -1,7 +1,9 @@
 import numpy as np
+import copy
 
 
 def _to_numpy(hinput, read_sumw2=False):
+    hinput = copy.deepcopy(hinput)
```

Comment:
Might make more sense to copy at the end of the function when the data is normalized to a tuple of numpy arrays.

Reply:
Can leave for another PR